src/account/saga.ts

```diff
@@ -220,6 +227,21 @@ export function* handleUpdateAccountRegistration() {
   }
 }
 
+export function* updateStatsigAndNavigate(action: UpdateStatsigAndNavigateAction) {
+  // Wait for wallet address to exist before updating statsig user
+  yield call(getWalletAddress)
+  yield call(patchUpdateStatsigUser)
+  if (action.screen === Screens.WalletHome) {
+    navigateHome()
+  } else if (action.screen) {
```

Comment:
```suggestion
  } else {
```
we don't need this if since it's required now right?

Reply:
just wasn't sure if this would fall through to `navigate` after calling `navigateHome` if we're going to the home screen